test/verify/machineslib.py

```diff
@@ -201,6 +201,15 @@ def tearDown(self):
         # which need to be adjusted before we can do that.
         pass
 
+    def addIface(self, name):
+        self.machine.execute(r"""
+            mkdir -p /run/udev/rules.d/ &&
+            echo 'ENV{ID_NET_DRIVER}=="veth", ENV{INTERFACE}=="%s", ENV{NM_UNMANAGED}="0"' > /run/udev/rules.d/99-nm-veth-test.rules &&
+            udevadm control --reload &&
+            ip link add name %s type veth
+            """ % (name, name))
```

Comment:
This is a bit ugly. With `%` you can use `"%(name)s" % { "name": name }`, or use `.format() with `{0}` twice.

Reply:
`{0}` does not work as there are other `{...}` in the string. But using `{ "name": name }` is a good idea.